.gitignore

```diff
@@ -1,10 +1,17 @@
-.vs/
```

Comment:
I don't think .vs or .vscode should be unignored -- you can just exempt the one settings.json file like:
```
!.vscode/settings.json
```